src/drivers/pgsql/drv_pgsql.c

```diff
@@ -112,6 +113,8 @@ static char use_ps; /* whether server-side prepared statemens should be used */
 
 /* PgSQL driver operations */
 
+
+static char** str_split(char *src, char sep, int *numparts);
```

Comment:
Why not use strtok?

Reply:
There are extra libraries required for this. I infact I used one of the library functions earlier and it started giving issues in ubuntu environment. Just to save ourselves the overhead of testing it with different environments, I moved away from library functions.  https://github.com/yugabyte/sysbench/pull/9/commits/c630e99da2a3d5ee2ab2ee4e702edc5df68db899